src/app/components/code-block/index.html

```diff
@@ -0,0 +1,54 @@
+<sky-demo-page
+  pageTitle="Code block"
+  skyuxModule="SkyCodeBlockModule"
+  packageName="lib-code-block"
+>
+
+  <sky-demo-page-summary>
+    <p>
+      The code block component formats a block of text as a code snippet with syntax highlighting. The code block also has a header that displays a copy button and, if defined, the language type. To create a code snippet, wrap <stache-code>sky-code-block</stache-code> tags around the code block.
+    </p>
+    <p>
+      If a code block has text within <stache-code>&lt;</stache-code> and <stache-code>&gt;</stache-code> characters, then the component automatically appends a closing tag unless the code snippet is valid HTML. To prevent this, you must use "&lt;" to escape <stache-code>&lt;</stache-code> and "&gt;" to escape <stache-code>&gt;</stache-code>. For example, you must escape the <stache-code>&lt;</stache-code> and <stache-code>&gt;</stache-code> characters in the code snippet <stache-code>public Task&lt;Generic> MethodName()</stache-code> or else the component automatically appends <stache-code>&lt;/Generic></stache-code> to the end: <stache-code>public Task&lt;Generic> MethodName()&lt;/Generic></stache-code>.
```

Comment:
The examples here don't work. The escape characters as presented in the "To prevent" sentence actually convert to the angle brackets they represent. I looked back in the Stache docs, and we need to replace "&lt;" with "&#38;lt;" and replace "&gt;" with "&#38;gt;"

Reply:
Looks like this was addressed, so I'm resolving the comment.